obm-autoset/README.md

```diff
@@ -0,0 +1,46 @@
+#RackHD OBM Setting Auto Set Tool
+Tool to apply OBM setting for each compute node discovered by RackHD ora. Save the trouble to manually apply OBM setting node by node.

```

Comment:
Suggest to remove the word "ora", I don't think every user is aware what does this word mean especially you are using the short name not the full name. I think use "RackHD" is enough here and at following content.